apollo-router/src/plugins/include_query_plan.rs

```diff
@@ -0,0 +1,241 @@
+use futures::future::ready;
+use futures::stream::once;
+use futures::StreamExt;
+use http::HeaderValue;
+use serde_json_bytes::json;
+use tower::util::BoxService;
+use tower::BoxError;
+use tower::ServiceExt as TowerServiceExt;
+
+use crate::layers::ServiceExt;
+use crate::plugin::Plugin;
+use crate::plugin::PluginInit;
+use crate::register_plugin;
+use crate::services::QueryPlannerContent;
+use crate::services::QueryPlannerRequest;
+use crate::services::QueryPlannerResponse;
+use crate::services::RouterRequest;
+use crate::services::RouterResponse;
+
+const INCLUDE_QUERY_PLAN_HEADER_NAME: &str = "X-Apollo-Query-Plan";
+const INCLUDE_QUERY_PLAN_ENV: &str = "APOLLO_INCLUDE_QUERY_PLAN";
```

Comment:
Since this just _enables_ the ability to use the header, perhaps something like:

```suggestion
const ENABLE_EXPOSE_QUERY_PLAN_ENV: &str = "APOLLO_ENABLE_EXPOSE_QUERY_PLAN";
```

or

```suggestion
const ENABLE_EXPOSE_QUERY_PLAN_ENV: &str = "APOLLO_EXPOSE_QUERY_PLAN";
```

?